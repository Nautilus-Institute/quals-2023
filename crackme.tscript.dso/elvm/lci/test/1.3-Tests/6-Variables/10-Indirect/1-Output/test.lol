HAI 1.3
	I HAS A var1 ITZ 1
	I HAS A var2 ITZ 2.345
	I HAS A var3 ITZ "67890"
	I HAS A name ITZ "var"
	VISIBLE SRS "var1"
	VISIBLE SRS SMOOSH name AN SUM OF 1 AN 1 MKAY
	VISIBLE SRS ":{name}3"
KTHXBYE
