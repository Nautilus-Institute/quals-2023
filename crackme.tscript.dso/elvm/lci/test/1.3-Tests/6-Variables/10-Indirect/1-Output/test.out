1
2.34
67890
